### ISSUE TITLE
Cannot use reverse tethered device as a hotspot. 

### ISSUE THREAD

**Issue by Lou-Snowfang** [Genymobile/gnirehtet#267]
The following is my command prompt log after using Rust version 2.4 on Windows 10. 

2020-02-18 15:48:00.951 INFO TunnelServer: Client #0 connected
2020-02-18 15:48:01.127 INFO UdpConnection: 10.0.0.2:46503 -> 8.8.8.8:53 Open
2020-02-18 15:48:02.449 INFO UdpConnection: 10.0.0.2:49604 -> 8.8.8.8:53 Open
2020-02-18 15:48:11.138 INFO UdpConnection: 10.0.0.2:46916 -> 8.8.8.8:53 Open
2020-02-18 15:48:12.460 INFO UdpConnection: 10.0.0.2:49579 -> 8.8.8.8:53 Open
2020-02-18 15:48:22.470 INFO UdpConnection: 10.0.0.2:44309 -> 8.8.8.8:53 Open
2020-02-18 15:48:32.480 INFO UdpConnection: 10.0.0.2:39686 -> 8.8.8.8:53 Open
2020-02-18 15:48:32.656 INFO UdpConnection: 10.0.0.2:49025 -> 8.8.8.8:53 Open
2020-02-18 15:48:42.666 INFO UdpConnection: 10.0.0.2:48667 -> 8.8.8.8:53 Open
2020-02-18 15:48:45.247 INFO TcpConnection: 10.0.0.2:48303 -> 149.154.175.50:5222 Open
2020-02-18 15:49:00.251 WARN Router: Dropping invalid packet
2020-02-18 15:49:04.528 WARN Router: Dropping invalid packet
2020-02-18 15:49:06.253 INFO TcpConnection: 10.0.0.2:48303 -> 149.154.175.50:5222 Close
2020-02-18 15:49:12.684 INFO UdpConnection: 10.0.0.2:41590 -> 8.8.8.8:53 Open
2020-02-18 15:49:12.720 INFO TcpConnection: 10.0.0.2:38721 -> 108.177.10.188:5228 Open
2020-02-18 15:49:13.013 INFO UdpConnection: 10.0.0.2:39174 -> 8.8.8.8:53 Open
2020-02-18 15:49:13.057 INFO TcpConnection: 10.0.0.2:40424 -> 172.217.12.74:443 Open
2020-02-18 15:49:13.319 WARN Router: Dropping invalid packet
2020-02-18 15:49:14.309 WARN Router: Dropping invalid packet
2020-02-18 15:49:19.132 INFO TcpConnection: 10.0.0.2:40425 -> 172.217.12.74:443 Open
2020-02-18 15:49:20.311 WARN Router: Dropping invalid packet
2020-02-18 15:49:21.310 WARN Router: Dropping invalid packet
2020-02-18 15:49:44.313 WARN Router: Dropping invalid packet
2020-02-18 15:49:45.305 WARN Router: Dropping invalid packet
2020-02-18 15:49:46.293 WARN Router: Dropping invalid packet
2020-02-18 15:49:49.305 WARN Router: Dropping invalid packet
2020-02-18 15:49:51.351 WARN Router: Dropping invalid packet
2020-02-18 15:49:53.315 WARN Router: Dropping invalid packet

Reverse tethered device is also not getting any kind of connection, despite being firmly connected by cable.

**Comment by Lou-Snowfang** [comment]
The main PC has a solid Ethernet connection, but isn't passing it to the phone.

**Comment by rom1v** [comment]
Is 8.8.8.8 (the google dns) accessible from your PC? Do you need a proxy on your PC?

**Comment by Lou-Snowfang** [comment]
Yes. The Google DNS server is accessible from my network. I do not use a
proxy.

**Comment by rom1v** [comment]
> as a hotspot

#95
